README.md

```diff
@@ -65,13 +66,14 @@ The World Service front pages follow the article format for AMP too, being avail
 - [http://localhost:7080/igbo.amp](http://localhost:7080/igbo.amp)
 - [http://localhost:7080/pidgin.amp](http://localhost:7080/pidgin.amp)
 
-
 ### Storybook (UI Development Environment/Style Guide)
 
 We use Storybook for developing components in isolation from the Simorgh Application. You can access this at [https://bbc.github.io/simorgh/](https://bbc.github.io/simorgh/)
 
 To run locally `npm run storybook`, it will then be available at [http://localhost:9001/](http://localhost:9001/). Introduction to and documentation for Storybook is here: [https://storybook.js.org/basics/introduction/](https://storybook.js.org/basics/introduction/).
 
+When viewing Video stories locally, make sure to use a BBC domain, as outline in the changing request location section. Video will not work in the hosted version of Storybook linked above for this reason.
```

Comment:
You should document _how_ to do this

Reply:
There's a section further down the page:  https://github.com/bbc/simorgh#changing-request-location

Should I link it directly?

Reply:
Yeah that'd be nice